src/components/Calculator.js

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
 import './Calculator.css';
 
 function Button(props) {
-  const { text, className } = props;
+  const { text, className, handleClick } = props;
```

Comment:
[OPTIONAL] To fully achieve the goal of separation of concerns, which is the reason we make use split our components in react, I suggest you take this `button` component to an independent file and import it here in the calculator component😉.

Reply:
Okay, thanks a lot for your kind feedback. I will do that as you suggested dear sir. Thanks again